numpy/_core/src/umath/string_buffer.h

```diff
@@ -895,10 +895,12 @@ string_find(Buffer<enc> buf1, Buffer<enc> buf2, npy_int64 start, npy_int64 end)
     npy_intp pos;
     switch(enc) {
         case ENCODING::UTF8:
-            pos = fastsearch(start_loc, end_loc - start_loc, buf2.buf, buf2.after - buf2.buf, -1, FAST_SEARCH);
+            pos = fastsearch(start_loc, end_loc - start_loc, buf2.buf,
+                    buf2.after - buf2.buf, -1, FAST_SEARCH);
```

Comment:
Strictly speaking, this line should line up with the arguments above. Same below.

Reply:
Running .clang-format on this file will probably fix that up, and maybe do some extra stuff you may want to undo.

Reply:
Uhhhh, running the `.clang-format` can sometimes lead to changes that violate the C-style guide, such as altering the formatting of all methods within a struct :
```c++
// origin code
inline Buffer()
{
    buf = after = NULL;
}
// after running .clang-format
inline Buffer() { buf = after = NULL; }
```
To be honest, I'm unsure how to configure `.clang-format` to align with our C-style guide.

Reply:
clang-format is sort of the replacement for the C-style guide.

Reply:
In case you are wondering, the indentation should look like this:
```
            pos = fastsearch(start_loc, end_loc - start_loc, buf2.buf,
                             buf2.after - buf2.buf, -1, FAST_SEARCH);
```
The other indentation is only when there are no arguments included in the first line. I have made exceptions to that, but mostly for exception messages. I'm not trying to be difficult here, but if you are working on style you need to become familiar with the conventions.

Reply:
> I'm not trying to be difficult here

I know, I appreciate your feedback, both of our goal is to enhance code readability. 
My change stems from my own difficulties in reading the code, and I hope to help others avoid similar issues.

> In case you are wondering, the indentation should look like this:

But according to the [C-Style Guide](https://numpy.org/neps/nep-0045-c_style_guide.html), it says that "It isn’t necessary to line everything up with the opening parenthesis of the function call." 
![image](https://github.com/user-attachments/assets/f54b5eb2-2be6-4e1e-bba8-dccf2180c4d5)

And in some cases (maybe the function's or param's name is too long or the amount of retract is a bit large), align the first parameter of each row could lead to several line break, so I agree with our C-Style Guide.